'oldValue' in type 'NgxStorageEvent<T>' is not assignable to the same property in base type 'StorageEvent'

Not sure if this is already being tracked, but got this error during build after installing latest version (`"ngx-store": "^3.0.0"`)
```
ERROR in node_modules/ngx-store/lib/utility/storage/storage-event.d.ts:6:5 - error TS2416: Property 'oldValue' in type 'NgxStorageEvent<T>' is not assignable to the same property in base type 'StorageEvent'.
  Type 'T' is not assignable to type 'string'.
```
For now i'm setting `"skipLibCheck": true,` but thats not ideal.

--- a/package.json
+++ b/package.json
@@ -61,7 +61,8 @@
     "@types/lodash.merge": "^4.6.4",
     "@types/lodash.set": "^4.3.4",
     "@types/node": "^14.14.36",
-    "ts-debug": "^1.3.0"
+    "ts-debug": "^1.3.0",
+    "tslint": "6.1.3"
   },
   "engines": {
     "node": ">= 8",

--- a/src/lib/utility/storage/storage-event.ts
+++ b/src/lib/utility/storage/storage-event.ts
@@ -1,8 +1,6 @@
-export class NgxStorageEvent<T = any> implements StorageEvent {
+export class NgxStorageEvent<T = any> implements Omit<StorageEvent, 'oldValue' | 'newValue'> {
   protected static initTimeStamp = Date.now();
-  // @ts-ignore
   public oldValue: T;
-  // @ts-ignore
   public newValue: T;
   public NONE: any;
   public timeStamp = (Date.now() - NgxStorageEvent.initTimeStamp);

